src/test/ui/borrowck/borrowck-mut-borrow-linear-errors.stderr

```diff
@@ -25,7 +25,10 @@ error[E0499]: cannot borrow `x` as mutable more than once at a time
   --> $DIR/borrowck-mut-borrow-linear-errors.rs:12:30
    |
 LL |             _ => { addr.push(&mut x); }
-   |                              ^^^^^^ `x` was mutably borrowed here in the previous iteration of the loop
+   |                    ----------^^^^^^-
+   |                    |         |
+   |                    |         `x` was mutably borrowed here in the previous iteration of the loop
+   |                    that borrow lives into the next loop iteration, preventing another mutable borrow because that borrow is used here
```

Comment:
```suggestion
   |                    that borrow lives into the next loop iteration, preventing another mutable borrow
```
I think the end of this label just increases verbosity and isn't more instructive.

Reply:
I think the use of the mutable borrow is reason of that borrow lives into the next iteration. If the borrow is not used, or is used only for read, there will be no problem at the next iteration. So I think that we need to point out that "that borrow is used here" is the reason of the real problem (the reference is used here so that it cannot be invalidated at next iteration), not because of the borrow over loop iteration. This can give the user more clear hint how to fix the error.